backend/docs/documentation.yaml

```diff
@@ -295,6 +305,70 @@ paths:
                       count_matches: 3
                     }
                   ]
+
+  /employer:
+    post:
+      description: Allows the user to save their employer profile on the database
+      tags:
+        - "Employer"
+      requestBody:
+        description: a json with all the needed informations about the employer
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              example:
+                {
+                  name: "Samsung",
+                  location: {
+                    country: "South Korea",
+                    city: "Seul"
+                  },
+                  description: "A big tech company",
+                  domainOfActivityId: 3
```

Comment:
This does not make sense with what you told me to do in your first comment in the DAO: you told me not to do a query to get the domain of activity in the employer creation function because that response should only confirm that the profile has been created, so now this doesn't make so much sense now that I deleted that query. Also we are using underscores everywhere in the responses body (while we use camel notation in the request body) because that's what is used in the DB, so it has to be "domain_of_activity".

Reply:
When I wrote this comment, I forgot that I said that you should not return the domain of the activity, however, the schema can contain it since it appears in the "body" of of the request. The issue with underscore is way larger, and will have to be tackled in another task, basically we should NOT return "x_y", we should use camel case in all the code, and only in the db we should have "x_y". Anyway, I left this comment because we need to start aligning things to a schema, and have the objects that we are sending and the responses adhere to it. 

We can leave it like this for now and we will change them in the refactoring task. Thanks!